### PR TITLE
Make host pvt subnet configurable

### PR DESCRIPTION
**Pull request by jojimt**
Fixes https://github.com/contiv/ccn/issues/186

### CODE REVIEW

**Comment by jojimt** [comment]
@abhinandanpb @gkvijay @rchirakk PTAL

File: netplugin/agent/state_event.go [hunk 1]
```diff
@@ -289,10 +308,13 @@ func processGlobalConfigUpdEvent(netPlugin *plugin.NetPlugin, opts core.Instance
 		},
 		Instance: opts,
 	}
-	pluginConfig.Instance.ArpMode = cfg.ArpMode
+	pluginConfig.Instance.ArpMode = arpMode
+	if pluginConfig.Instance.FwdMode == "routing" && arpMode == "flood" {
+		log.Infof("Global ARP mode config is not effective when forwarding mode is routing. Proxy-arp will be retained.")
```
Comment (abhinandanpb) [inline netplugin/agent/state_event.go]: should we make this printf ? If we want the use to know when he/she configures ?

File: netplugin/agent/state_event.go [hunk 2]
```diff
@@ -289,10 +308,13 @@ func processGlobalConfigUpdEvent(netPlugin *plugin.NetPlugin, opts core.Instance
...
+	}
 	netPlugin.GlobalConfigUpdate(pluginConfig)
 
-	log.Infof("Global Config updated")
+	log.Infof("ARP mode updated")
```
Comment (abhinandanpb) [inline netplugin/agent/state_event.go]: minor nit - can also mention updated to flood/proxy.

**Comment by abhinandanpb** [comment]
LGTM

File: netplugin/agent/state_event.go (hunk 1 above)
Comment (jojimt) [inline netplugin/agent/state_event.go]: Moving this to printf will not help as this is in the netplugin logs. I think we can consider printing a similar message when the object is updated.

**Pull request merged**